third_party/amd/lib/TritonAMDGPUToLLVM/ElementwiseOpToLLVM.cpp

```diff
@@ -1494,6 +1494,58 @@ struct ExpOpConversionApprox
   }
 };
 
+class ClampFOpConversion : public ClampFOpConversionBase {
```

Comment:
one alternative is splitting this pattern into a generic one and a target specific optimized one with higher benefits. what do you think?